VirtusLab-Open-Source/strapi-plugin-navigation
# feat: strapi v5 rc

## Ticket

No ticket

## Summary

Migration of the plugin to Strapi V5

## Test Plan

- prepare strapi instance with version 5
- clone this repo with this branch
- build with `yarn build`
- connect via plugins.ts config
- start strapi instance
- check for bugs :)
